crates/driver/src/infra/simulator/tenderly/dto.rs

```diff
@@ -25,22 +25,14 @@ pub struct Request {
 
 #[derive(Debug, Deserialize)]
 pub struct Response {
-    transaction: Transaction,
-    generated_access_list: Option<AccessList>,
-}
-
-impl From<Response> for super::Simulation {
-    fn from(res: Response) -> Self {
-        Self {
-            gas: res.transaction.gas_used.into(),
-            access_list: res.generated_access_list.unwrap_or_default().into(),
-        }
-    }
+    pub transaction: Transaction,
+    pub generated_access_list: Option<AccessList>,
 }
 
 #[derive(Debug, Deserialize)]
-struct Transaction {
-    gas_used: u64,
+pub struct Transaction {
+    pub status: bool,
```

Comment:
I know this is outside our control but `status: true` is a really poor name.

Reply:
💯 

I _think_ it was chosen to [match the Ethereum JSON RPC API for transaction receipts](https://ethereum.github.io/execution-apis/api-documentation/#eth_getTransactionReceipt):

<img width="261" alt="image" src="https://github.com/cowprotocol/services/assets/4210206/cb800724-859c-4ddd-adbd-807934003f35">